Marlin/SanityCheck.h

```diff
@@ -261,6 +261,15 @@
 #endif
 
 /**
+ * DHT Sensor option
+ */
+#if ENABLED(DHT_ENABLE)
+  #if DISABLED(ULTIPANEL)
+    #error "DHT Sensor option requires an LCD controller."
+  #endif
+#endif
```

Comment:
How about this instead?

`#define DHT_ENABLE`
`#if ENABLED(DHT_ENABLE)`
    `#ifndef LCD_INFO_MENU`
      `#define LCD_INFO_MENU //This must be defined.`
    `#endif`
    `#define DHT_TYPE 22     //DHT type 11, 21, or 22`
    `#define DHT_PIN A3      //Set pin for the DHT sensor.`
`#endif //DHT_ENABLE`